types/anchor-js/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Without the extra rules the linter kept complaining about the `declare module` at the bottom but its recommendations weren't working for me. How should this be written?

```typescript
declare module 'anchor-js' {
    export = AnchorJS;
}
```

Reply:
try just `export = AnchorJS;` ?

Reply:
Doing that gives errors in the test file.
```
Error: c:/dev/GitHub/DefinitelyTyped/types/anchor-js/anchor-js-tests.ts:1:1
ERROR: 1:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 2:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 3:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 5:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 6:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 7:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 9:1    expect  TypeScript@next compile error:
Cannot find name 'anchors'.
ERROR: 11:27  expect  TypeScript@next compile error:
Cannot find name 'AnchorJS'.
ERROR: 13:27  expect  TypeScript@next compile error:
Cannot find name 'AnchorJS'.
```

Reply:
@andy-ms

Reply:
@andy-ms if I switch the file around to use that then I need to add `import { anchors, AnchorJS } from "anchor-js";` to the top of my test file. Is this correct? I can push my changes for review if it'd help.

Reply:
I think I read the file wrong -- `AnchorJS` is the one equivalent to the module. So you should write `export as namespace AnchorJS;`, and also `declare global { const anchors: anchorjs.Anchor; }` (how is that variable used from a module though?). Once the globals are declared you shouldn't need imports in the test file.